spec/pusher/push_notifications/generate_token_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'jwt'
+
+RSpec.describe Pusher::PushNotifications::UseCases::GenerateToken do
+  subject(:use_case) { described_class.new(user: user) }
+
+  let(:user) { 'Elmo' }
+  let(:instance_id) { ENV['PUSHER_INSTANCE_ID'] }
+  let(:secret_key) { ENV['PUSHER_SECRET_KEY'] }
+
+  describe '#generate_token' do
+    subject(:generate_token) { use_case.generate_token }
+
+    context 'when user id is empty' do
+      let(:user) { '' }
+
+      it 'token generation will fail' do
+        expect { generate_token }.to raise_error(
+          Pusher::PushNotifications::UseCases::GenerateToken::GenerateTokenError
+        ).with_message(
+          'User Id cannot be empty.'
+        )
+      end
+    end
+
+    context 'when user id is too long' do
+      let(:user) { 'a' * 165 }
```

Comment:
You can grab the length from the constant and then do a + 1. That means if you want to change the size, you'd only ever need to change 1 place and the tests would still pass.

Reply:
Fixed in 2d514d9.